aptos-move/block-executor/src/view.rs

```diff
@@ -960,20 +961,32 @@ impl<'a, T: Transaction, X: Executable> ViewState<'a, T, X> {
 /// all necessary traits, LatestView is provided to the VM and used to intercept the reads.
 /// In the Sync case, also records captured reads for later validation. latest_txn_idx
 /// must be set according to the latest transaction that the worker was / is executing.
-pub(crate) struct LatestView<'a, T: Transaction, S: TStateView<Key = T::Key>, X: Executable> {
+pub(crate) struct LatestView<
+    'a,
+    T: Transaction,
+    S: TStateView<Key = T::Key>,
+    X: Executable,
+    E: WithDeserializerConfig,
+> {
     base_view: &'a S,
+    #[allow(dead_code)]
```

Comment:
why is it dead?

Reply:
Removed, will let other PRs propagate it if needed.

Reply:
Removed all propagation of `E` to LatestView*